integration-tests/utils/integration-helpers.js

```diff
@@ -180,13 +180,17 @@ const withRegisteredUser = test => withUser(async options => {
   await test(options)
 })
 
+const overrideConfig = async (page, config) => {
+  await page.evaluate(passedInConfig => window.configOverridesStore.set(passedInConfig), config)
```

Comment:
nit: Is this more readable, or just cruft?

```suggestion
const overrideConfig = async (page, configToPassIn) => {
  await page.evaluate(passedInConfig => window.configOverridesStore.set(passedInConfig), configToPassIn)
```

Reply:
I don't feel strong either way so made the change in [4baa27c2](https://github.com/DataBiosphere/terra-ui/pull/2865/commits/4baa27c20c239185ec2a99c1fc6a69b2f8fc2024).